releases/8.0.0.md

```diff
@@ -71,12 +71,15 @@ Python:
 
 JavaScript:
     Upgraded:
-        primevue == 4.1.0
+        primevue == 4.2.5
+        vue ^3.5.13
 
     Added:
         primeicons == 7.0.0
+        @primeuix/themes == 1.01
```

Comment:
```
$ npm show @primevue/themes@4.0.7
@primevue/themes@4.0.7 | MIT | deps: 1 | versions: 25
https://primevue.org/
DEPRECATED :warning:  - This package is deprecated. Use @primeuix/themes instead.
```

Migrating away from `@primevue/themes` should prove less fragile -- we were seeing problems when upgrading primevue and not upgrading the sidecar themes package to the same exact version.